### PR TITLE
Script that calculate the scores of the teams. 

### PR DESCRIPTION
**Pull request by matheuscanhizares**
A results.json added as an example.

### CODE REVIEW

File: _data/results.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+"round-1":[
```
Comment (charly3pins) [inline _data/results.json]: Please follw the same schema for results and teams and scores, with hyphon - or without, but not two different ways

File: _data/results.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  {
+    "team1": "Equipo 1",
+    "score1": 2,
+    "team2": "Equipo 2",
+    "score2": 1
+  },
```
Comment (charly3pins) [inline _data/results.json]: if we'll do the pichcichi classifiction will be interestinh have a structure to save them too inside team

File: _data/results.json (hunk 1 above)
Comment (matheuscanhizares) [inline _data/results.json]: Round-1 is the name of the round and team1, score1... is the variable but OK I will put in the same way.

File: _data/results.json (hunk 2 above)
Comment (matheuscanhizares) [inline _data/results.json]: Sure. But first we will create the script for the classification and then we will focus on the pichichi

**Pull request merged**